src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private LinkedList<Entry<K, V>>[] buckets;
+    private int size;
+
+    public MyHashMap() {
+        this(DEFAULT_CAPACITY);
+    }
+
+    public MyHashMap(int initialCapacity) {
+        buckets = new LinkedList[initialCapacity];
+        size = 0;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int bucketIndex = (key == null) ? 0 : getBucketIndex(key);
```

Comment:
```suggestion
        int bucketIndex = getBucketIndex(key);
```
Redundant  conditional